Update iOS offline map instructions

elliott-liu opened the pull request:
There seems to be a lot of confusion ([1](https://meshtastic.discourse.group/t/offline-maps-on-meshtastic/137/37), [2](https://meshtastic.discourse.group/t/offline-map-data/10504/4), [3](https://www.reddit.com/r/meshtastic/comments/17h2e58/ios_app_how_to_force_the_app_to_download_and_only/)) based on the previous instructions for offline maps.

I've gone ahead and updated the instructions with working instructions of the current implementation, and moved the previous instructions under "Legacy Mesh Map".

I'm not 100% sure it was app version 2.2.10 that added the toggle  but going through the commit history this is the earliest known mention of the term.

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/meshtastic/meshtastic?pullRequest=1352) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/meshtastic/meshtastic?pullRequest=1352) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/meshtastic/meshtastic?pullRequest=1352) it.</sub>

garthvh reviewed (commented):
Mbtiles support is gone, the offline tiles will remain until iOS 18 is released

elliott-liu commented:
As far as I can tell the default settings of the iOS app is not using .mbtiles. To use .mbtiles you need to toggle on "Legacy Mesh Map" in Settings.

I've moved the original .mbtile instructions under a heading "Legacy Mesh Map" to make things clearer, and added instructions for the default setting (using offline Apple Maps).

I was hoping this would clear up a some confusion, or at least make things a bit more clearer.

elliott-liu commented:
Once iOS 18 is released .mbtiles the "Legacy Mesh Map" in the documentation can be removed or adjusted as needed.

garthvh reviewed (commented):
Not iOS 18 dependent, was removed yesterday
